client/themes/morning.css

```diff
@@ -46,8 +46,22 @@ body {
 	padding: 15px 16px;
 }
 
+/* stylelint-disable */
 #chat .search::-webkit-input-placeholder {
-	color: #99a2b4;
+/* stylelint-enable */
+	color: #99a2b4 !important;
+	opacity: .5;
+}
+
+/* stylelint-disable */
+#chat .search::-moz-placeholder {
+/* stylelint-enable */
```

Comment:
Good theme management means user themes are stored outside of the lounge source (code) folder. And that by default means it won't be built/linted/whatever. Linting is a dev tool anyway.